packages/url/src/index.js

```diff
@@ -170,6 +170,11 @@ export function isValidFragment( fragment ) {
  * @return {string} URL with arguments applied.
  */
 export function addQueryArgs( url = '', args ) {
+	// If no arguments are to be appended, return original URL.
+	if ( ! Object.keys( args ).length ) {
```

Comment:
Watching the code it seems a check for null or undefined args would fit well here `if ( ! args || ! Object.keys( args ).length ) {`. But having a null or undefined value passed to this function is probably a bug in other place and we would just hide it. I'm not sure what is better here.

Reply:
I think it'd be fine to anticipate this, or at least if we wanted to not support it, we should probably throw our own custom `TypeError`, rather than allow an uncaught error to be thrown from the `Object.keys`.

Reply:
Funny enough, this is actually what's causing the E2E test failure:

1. https://github.com/WordPress/gutenberg/blob/232012c141da400d367e59199da84edfbeec87bb/packages/editor/src/components/post-permalink/index.js#L130
2. https://github.com/WordPress/gutenberg/blob/232012c141da400d367e59199da84edfbeec87bb/packages/editor/src/utils/url.js#L21-L23

Reply:
Should be fixed with d1513b3